src/Bridge/Symfony/HttpFoundation/RequestFactory.php

```diff
@@ -21,6 +21,9 @@ public function make(SwooleRequest $request): HttpFoundationRequest
             $server['HTTP_'.\mb_strtoupper(\str_replace('-', '_', $key))] = $value;
         }
 
+        $queryString = $server['QUERY_STRING'] ?? '';
```

Comment:
Are you certain that swoole sets `$server['QUERY_STRING']` ?

Reply:
yes, proof is here: https://github.com/swoole/swoole-src/blob/master/swoole_http_request.cc#L185

and here: https://github.com/swoole/swoole-src/blob/master/swoole_http_request.cc#L320